### PR TITLE
only run semgrep pro tests if repo is owned by returntocorp

### PR DESCRIPTION
**Pull request by tpetr**
will test by forking this branch

### CODE REVIEW

**Review by aryx — APPROVED** [review]

**Comment by spencerdrak** [comment]
@tpetr thanks for opening this up - my only suggestion here would be to add another trigger to the job so that it also runs on develop. Not sure the performance implications of doing this, but, it might be a nice safety net so that we know which changes caused issues and aren't getting unrelated failures in follow-on PRs. 

Alternatively we can check this nightly, but I don't know what the team would prefer.

File: .github/workflows/test-semgrep-pro-latest.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
 jobs:
```
Comment (spencerdrak) [inline .github/workflows/test-semgrep-pro-latest.yaml:13]:
```suggestion
  push:
    branches:
      - "develop"
jobs:
```

**Review by spencerdrak — COMMENTED** [review]
I'll defer to @aryx or @brandonspark on what they'd prefer to see here, but this should be able to run just fine on develop.

File: .github/workflows/test-semgrep-pro-latest.yaml (hunk 1 above)
Comment (spencerdrak) [reply .github/workflows/test-semgrep-pro-latest.yaml:13]: Also haven't tested this explicitly, so @tpetr may need to run another test from his fork

**Comment by spencerdrak** [comment]
@tpetr thanks for opening this up - my only suggestion here would be to add another trigger to the job so that it also runs on develop. Not sure the performance implications of doing this, but, it might be a nice safety net so that we know which changes caused issues and aren't getting unrelated failures in follow-on PRs. 

Alternatively we can check this nightly, but I don't know what the team would prefer.

File: .github/workflows/test-semgrep-pro-latest.yaml (hunk 1 above)
Comment (brandonspark) [reply .github/workflows/test-semgrep-pro-latest.yaml:13]: i see no reason not to do this, so i'm down

**Review by aryx — APPROVED** [review]

**Pull request merged**